src/batch-submitter/state-batch-submitter.ts

```diff
@@ -193,7 +206,10 @@ export class StateBatchSubmitter extends BatchSubmitter {
       'appendStateBatch',
       [batch, startBlock]
     )
-    while (tx.length > this.maxTxSize) {
+    while (tx.length / 2 > this.maxTxSize) {
```

Comment:
@karlfloersch this is the only real logic change i made - looks like you're trying to compare size, in which case we should divide by 2 right?

Reply:
The division by 2 is a good bug catch, it needs to take into account the `0x` prefix.

I wish that ethereum libraries used `Buffer` instead of `string` for things so that this wasn't the case